site-admin.md

```diff
@@ -0,0 +1,499 @@
+
+The codebase takes advantage of [Vercel environment
+variables](https://vercel.com/docs/concepts/projects/environment-variables) at
+build time. For example, the preview mode overlay links to the git branch that
+the site was built from (when that environment variable is available at build
+time).
+
+## Ways to view the site at different stages 🔍
+
+Throughout the develop-deploy process, there are several ways to view the
+website. The following table summarizes the important ways in which those views
+differ from each other.
+
+| Name                        | How to access          | GitHub branch  | Vercel env | Storyblok API key | Next.js preview? | Storyblok version param | Display top overlay? | Top overlay color | Button to enter/exit preview? |
+| --------------------------- | ---------------------- | -------------- | ---------- | ----------------- | ---------------- | ----------------------- | -------------------- | ----------------- | ----------------------------- |
+| Production                  | .com/.org URL          | `main`         | production | public            | off              | `published`             | No                   | n/a               | n/a                           |
+| Storyblok (yellow overlay)  | via Storyblok UI       | `develop`      | preview    | preview           | on               | `draft`                 | Yes                  | yellow            | No                            |
+| Vercel URL (gray overlay)   | via link to Vercel URL | any non-`main` | preview    | preview           | off              | `published`             | Yes                  | gray              | Yes                           |
+| Vercel URL (yellow overlay) | via enter-preview      | any non-`main` | preview    | preview           | on               | `draft`                 | Yes                  | yellow            | Yes                           |
+
+Let's take the row labeled "Vercel URL (gray overlay)." This view is accessed by
+clicking on a Vercel SHA-style URL, which looks like
+`https://labs-{SHA}-quansight.vercel.app`. Typically, this URL is found
+either on a GitHub pull request or in `#website-vercel-bot-log` in Slack. The site served by that URL can be
+built from any branch or commit on GitHub except `main`, which is reserved for
+production. It is built with the "preview" Vercel environment. It does not start
+in Next.js preview mode (though the end user can switch to it). The preview
+Vercel environment contains the preview Storyblok API key. The site passes
+`version=published` to the Storyblok API. It displays a gray overlay at the top
+of each page, and that gray overlay contains a way for the end user to switch to and out of the Next.js preview mode.
+
+Note that within Storyblok, there is no button to switch out of preview mode.
+This was done by design because the whole point of the Storyblok UI is to be
+able to preview content that hasn't been published yet.
+
+## System design decisions 🏗
+
```

Comment:
```suggestion

> [!CAUTION]
> This section is outdated. Blog posts are now merged directly to the `main` branch, and Storyblok updates are also directly published to the production website.
> These updates were made to streamline the website update workflow.

```

This section will need some major changes, so adding an admonition as a quick update.